README.md

```diff
@@ -13,6 +13,7 @@
 <a href="https://pepy.tech/project/modin"><img src="https://static.pepy.tech/personalized-badge/modin?period=total&units=international_system&left_color=black&right_color=blue&left_text=Downloads" align="center"></a>
 <a href="https://codecov.io/gh/modin-project/modin"><img src="https://codecov.io/gh/modin-project/modin/branch/master/graph/badge.svg" align="center"/></a>
 <a href="https://github.com/modin-project/modin/actions"><img src="https://github.com/modin-project/modin/workflows/master/badge.svg" align="center"></a>
+<a href="https://github.com/modin-project/modin/actions/workflows/ci.yml"><img src="https://github.com/modin-project/modin/actions/workflows/ci.yml/badge.svg?branch=master" align="center"></a>
```

Comment:
Why?

Reply:
As quite a few tests are removed from the Sanity subset, I think we need an obvious way to check for CI tests passing against `master` branch, hence a new badge.

Reply:
How is this badge different from the one above?

Reply:
The links are different. Upper one leads to all actions, and added one leads specifically to `ci` workflow status.